runtime/src/message_processor.rs

```diff
@@ -125,13 +125,13 @@ impl PreAccount {
 
         // An account not assigned to the program cannot have its balance decrease.
         if *program_id != pre.owner // line coverage used to get branch coverage
-         && pre.lamports > post.lamports
+         && pre.lamports() > post.lamports
         {
             return Err(InstructionError::ExternalAccountLamportSpend);
         }
 
         // The balance of read-only and executable accounts may not change
-        let lamports_changed = pre.lamports != post.lamports;
+        let lamports_changed = pre.lamports() != post.lamports;
```

Comment:
Does `post.lamports` need to be `post.lamports()`?

Reply:
maybe, will pick these up on a future pr when I do another batch of stragglers.